dialogflow_v2/gapic/intents_client.py

```diff
@@ -798,9 +792,9 @@ def batch_update_intents(
 
         request = intent_pb2.BatchUpdateIntentsRequest(
             parent=parent,
-            language_code=language_code,
             intent_batch_uri=intent_batch_uri,
             intent_batch_inline=intent_batch_inline,
+            language_code=language_code,
```

Comment:
`language_code` is now optional as expected

Reply:
WOO!